Firebase/DynamicLinks/CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
+# v4.0.3
+- [added] Added support for custom domains without FirebaseCore. (#3540)
```

Comment:
s/without FirebaseCore/for internal Google apps/

We don't support skipping FirebaseCore for external users.